doc/reference/hardware.rst

```diff
@@ -1,196 +1,97 @@
+.. _UP2 Shop:
+   https://up-shop.org/home/270-up-squared.html
+
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+-------------+-------------+
+|                        |                                    |                                  ACRN Version                                     |
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+-------------+-------------+
+| Intel Processor Family | Tested Product                     |   v1.0      |   v1.6.1    |    v2.0     |    v2.5     |    v2.6     |    v2.7     |
++========================+====================================+=============+=============+=============+=============+=============+=============+
+| Tiger Lake             | `NUC11TNHi5`_                      |             |             |             | Release     | Maintenance               |
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+---------------------------+
+| Whiskey Lake           | `WHL-IPC-I5`_                      |             |             | Release     | Maintenance                             |
++------------------------+------------------------------------+-------------+-------------+-------------+-------------+---------------------------+
+| Kaby Lake              | `NUC7i7DNHE`_                      |             | Release     | Maintenance               | Community Support         |
++------------------------+------------------------------------+-------------+-------------+---------------------------+---------------------------+
+| Apollo Lake            | `NUC6CAYH`_, `UP2-N3350`_,         | Release     | Maintenance | Community Support                                     |
+|                        | `UP2-N4200`_, `UP2-x5-E3940`_      |             |             |                                                       |
++------------------------+------------------------------------+-------------+-------------+-------------------------------------------------------+
+
+* **Release**: New ACRN features are complete and tested for the listed product.
+  This product is recommended for this ACRN version. Support for older products
+  will transition to the maintenance category as development continues for newer
+  products.
+
+* **Maintenance**: For new ACRN versions with maintenance-level support, we
+  verify our Getting Started Guide instructions to ensure the baseline
+  development workflow works and the hypervisor will boot on the listed
+  products. While we don't verify that all new features will work on
+  this product, we will do best-effort support on reported issues.
+
+* **Community Support**: Community responds with best-effort support for that
+  ACRN version to reported bugs for the listed product.
+
+To begin your exploration of ACRN, we recommend selecting a product from the

```

Comment:
Instead of this paragraph, how about something like this:
```
When you start to explore ACRN, we recommend you select
the latest product from the table above. You can also choose
other products and give them a try.  In either case, use the
:ref:`board_inspector_tool>` to generate a board configuration file
you will use to configure the ACRN hypervisor, as described in the
:ref:`gsg`.  We encourage your feedback on the
acrn-user@lists.projectacrn.org mailing list on your findings about
unlisted products.
```